pyproject.toml

```diff
@@ -27,7 +27,7 @@ janus = "janus_core.cli.janus:app"
 
 [tool.poetry.dependencies]
 python = "^3.9"
-ase = "^3.22.1"
+ase = "3.22.1"
```

Comment:
if you fix this there is no FrechetCellFilter... and will simplify things around the default filter.. but of course we will reset this after.

Reply:
This is just from rebasing onto the `pin-ase` branch (#177).

Until the most recent tag, it was necessary to overwrite this line to get the `FrechetCellFilter` anyway, so this just maintains the previous behaviour until fixes from #178 can be implemented, otherwise tests will fail.

Reply:
I wanted to avoid to have it hardcoded to LBFGS somewhere...

Reply:
Sorry, I'm not sure I understand what you're proposing?

Reply:
Surely that's still hardcoded, just with less accessible help? What's the advantage?

Also, is this still relevant to the commented line change?

```
- ase = "^3.22.1"
+ ase = "3.22.1"
```